content/docs/release-notes/2305-0-release-notes.md

```diff
@@ -0,0 +1,202 @@
+---
+linktitle: 23.05.0 Release Notes
+title: Release Notes for Open 3D Engine 23.05.0
+description: Full release notes for Open 3D Engine (O3DE) version 23.05.0.
+weight: 895
+toc: true
+---
+
+## Highlights for 23.05.0
+
+Here are some highlights of **Open 3D Engine (O3DE)** 23.05.0, followed by a detailed list of features broken down by SIG.
+
+1.	New and Improved Authoring Experiences: 
+Material Canvas is a new intuitive, easy to use, node based, visual scripting interface that allows users to create new, custom shaders and materials. It is built upon the combined foundations of script canvas and Material Editor that O3DE users are already familiar with. For more information, refer to [sig-graphics-audio#51](https://github.com/o3de/sig-graphics-audio/issues/51).
+O3DE now has a Material Pipeline, a layer of abstraction between lighting and materials which utilizes new builders and new features from the asset system to build unique shaders for each render pipeline. This provides modularity and extensibility to the Atom Renderer, allowing easier customization of the render pipeline.
+Physics & animations improvements including support for PhysX 5.1 and improvements in multiple PhysX authoring workflows. Users will notice an update to the Animation Editor experience, which is now more consistent with other O3DE tools. The Animation Asset Import process is now more robust and straightforward, and the Animation Editor AnimGraph now has a Performance Visualizer to help users profiling and optimizing their AnimGraphs.
+Terrain System now provides users a paintbrush tool so they may paint inside the viewport to create or modify terrain.
+
+2.	Multiplayer Sample Game: The [`o3de-multiplayersample` repository](https://github.com/o3de/o3de-multiplayersample) is home to O3DE's newest project, a networked, player versus player versus environment game that supports 1 to 15 players. In this game, players compete for the highest score to win. Over a series of rounds, players race around the starbase to collect gems and rack up points. Each player is armed with a laser pistol and protected by a shield. Taking damage from laser blasts depletes the player's shield. When the shield is depleted, the player respawns at the cost of some of their collected gems. Do you risk it all to win? For more information, refer to the [Multiplayer Sample Project README](https://github.com/o3de/o3de-multiplayersample#readme).
```

Comment:
```suggestion
2.	Multiplayer Sample Game: The [`o3de-multiplayersample` repository](https://github.com/o3de/o3de-multiplayersample) is home to O3DE's newest project, a networked, player versus player versus environment game that supports 1 to 10 players. In this game, players compete for the highest score to win. Over a series of rounds, players race around the starbase to collect gems and rack up points. Each player is armed with a laser pistol and protected by a shield. Taking damage from laser blasts or from the environmental hazards depletes the player's shield. When the shield is depleted, the player respawns at the cost of some of their collected gems. Do you risk it all to win? For more information, refer to the [Multiplayer Sample Project README](https://github.com/o3de/o3de-multiplayersample#readme).
```

Reply:
Fixed and edited out some of the detailed description as it's copied from the first paragraph of the linked README.